various minor changes to staking pages (#8692)
ethereum/ethereum-org-website: src/content/staking/solo/index.md

Solo staking is the act of [running an Ethereum node](/run-a-node/) connected to the internet and depositing 32 ETH to activate a [validator](#faq), giving you the ability to participate directly in network consensus.

> **Solo staking increases the decentralization of the Ethereum network**, making Ethereum more censorship-resistant and robust against attacks. Other staking methods may not help the network in the same ways. This is why solo staking is the gold standard for securing Ethereum.

Reviewer: ```suggestion
**Solo staking increases the decentralization of the Ethereum network**, making Ethereum more censorship-resistant and robust against attacks. Other staking methods may not help the network in the same ways. Solo staking is the best staking option for securing Ethereum.
```

> **Solo-staking increases the decentralization of the Ethereum network**, making Ethereum more censorship-resistant and robust against attacks. Other staking methods may not help the network in the same ways. This is why staking at home is the gold standard for securing Ethereum.

Reviewer: ```suggestion
**Solo-staking increases the decentralization of the Ethereum network**, making Ethereum more censorship-resistant and robust against attacks. Other staking methods may not help the network in the same ways. This is why solo staking is the gold standard for securing Ethereum.
```

> **Solo-staking increases the decentralization of the Ethereum network** and it makes Ethereum more censorship resistant and robust against attacks. Other staking methods may not help the network in the same ways. This is why staking at home is the gold standard for securing Ethereum.

Reviewer: Active voice

```suggestion
**Solo-staking increases the decentralization of the Ethereum network**, making Ethereum more censorship-resistant and robust against attacks. Other staking methods may not help the network in the same ways. This is why staking at home is the gold standard for securing Ethereum.
```

Reviewer: > Other staking methods may not help the network in the same ways.

I'd be a little more explicit here.

Reviewer: ^ i.e. should we be clearer that they could actually harm Ethereum?

## Why stake solo? {#why-stake-solo}

> Solo staking comes with more responsibility - this is why validators get paid in ETH - but provides you with maximum control over your funds and staking setup.

Reviewer: Don't all stakers get paid in ETH? This seems to imply otherwise.

Author: yes they do - i was trying to convey the point that the reason there is a return is because you perform some useful function and have some responsibility --> not free money.

Reviewer: But this is true of all forms of staking. Not strongly opposed, but I don't see the utility of this statement. 

If someone is looking at our solo staking page, I'd guess they are likely to know that staking is taking on some economic responsibility for returns.

Author: ```suggestion
Solo staking comes with more responsibility but provides you with maximum control over your funds and staking setup.
```
